README.md

```diff
@@ -1,6 +1,22 @@
 # Reddit
 Reddit clone for educational purposes. Developed by PyMasters
-Want to join and work on it? https://pymasters.pl/spolecznosc
+
+- [Joining the project](#joining-the-project)
+- [Rules of engagement](#rules-of-engagement)
+   - [Rules](#rules)
+   - [Code](#code)
+   - [Testing](#testing)
+      - [Writing own tests](#writing-own-tests)
+- [How to Set up](#how-to-set-up)
+   - [Connecting gmail to the application](#connecting-gmail-to-the-application)
+- [Ruff formatting and linting](#ruff-formatting-and-linting)
+- [Usage with Docker 🐳](#usage-with-docker-🐳)
+- [In case of problems with starting the project](#in-case-of-problems-with-starting-the-project)
```

Comment:
when I enter the link, it redirects me to the git page where there is no template, maybe it's just me.
![image](https://github.com/user-attachments/assets/96bc30c6-8949-437c-8239-4994238ffd00)